ros/openapi/openapi.json

```diff
@@ -341,6 +341,9 @@
                     "inventory_id": {
                         "type": "string"
                     },
+                    "display_name": {
```

Comment:
Along with `display_name`, needs to add `fqdn` property as well under `SystemDetails`schema.

Reply:
@kgaikwad the output for `/systems/<inventory_id>` is:
```json
{
    "inventory_id": "07d1e5ab-8f7b-41f6-ad8e-b0fd4d1e4fbf",
    "display_name": "ip-172-31-45-168.ap-south-1.compute.internal",
    "display_performance_score": {
        "cpu_score": 1,
        "memory_score": 3,
        "io_score": 1
    },
    "rating": 0,
    "number_of_recommendations": 1,
    "state": "Idling",
    "report_date": "2021-09-14",
    "instance_type": "t2.micro",
    "cloud_provider": "aws",
    "idling_time": "99.60",
    "io_wait": "0.00"
}
```
which does not include `fqdn`. Do we still want to add that in `SystemDetails` schema?

Reply:
@apuntamb It's there, I've added fqdn

Reply:
@r14chandra @kgaikwad Gotcha! I was referring to your older commit.